src/syscheckd/whodata/win_whodata.c

```diff
@@ -918,6 +991,113 @@ int whodata_audit_start() {
     return 0;
 }
 
+int policy_check() {
```

Comment:
Let's include more `debug2` messages in the flow of this function, to know what is checking and the results of the verifications.

Reply:
Done